Tests/StencilSwiftKitTests/XCTest+Assertions.swift

```diff
@@ -0,0 +1,12 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+
+public func XCTAssertThrows(_ expression: @autoclosure () throws -> Any?, _ message: @autoclosure () -> String = "",
```

Comment:
Could you instead move that into the `TestsHelper.swift` existing file?

(tbh I was sure we already implemented a similar helper somewhere but maybe it was in another PR?!)